controller/src/main.cpp

```diff
@@ -52,9 +52,16 @@ limitations under the License.
 #include "parameters.h"
 #include "pid.h"
 #include "sensors.h"
-#include "ventilator.h"
 #include "watchdog.h"
 
+static void ventilator_loop() {
```

Comment:
I'm wondering if we still want to call this "ventilator_loop"? This is the main controller loop, no?

Reply:
Renamed to `controller_loop`!